tools/indexer.py

```diff
@@ -11,7 +11,15 @@
 llama_index.set_global_handler("simple")
 
 # Select Model
-service_context = get_watsonx_context(model="ibm/granite-13b-chat-v1")
+## check if we are using remote embeddings via env
+tei_embedding_url = os.getenv("TEI_SERVER_URL", None)
+
+if tei_embedding_url != None:
+    service_context = get_watsonx_context(model="ibm/granite-13b-chat-v1", 
```

Comment:
Do we want to parameterize the other arguments in the function too? just a nit maybe we can do it in our subsequent PRs.

Reply:
If you're using the TEI server, technically it ignores the `tei_embedding_model` completely. The `get_watsonx_context` function already has parameterized stuff, so I'm not sure what you're asking.